src/mate/src/Command/ServeCommand.php

```diff
@@ -83,6 +84,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         );
 
         $server = Server::builder()
+            ->setProtocolVersion(ProtocolVersion::tryFrom($this->container->getParameter('mate.mcp_protocol_version')) ?? ProtocolVersion::V2025_03_26)
```

Comment:
if `getParameter` returns null, we might run in a deprecation here
https://3v4l.org/OPKGQ